Dockerfile

```diff
@@ -18,17 +18,19 @@ RUN apk add --no-cache --quiet \
 COPY package.json yarn.lock ./
 RUN yarn install --frozen-lockfile && yarn cache clean
 
-# Copy application code
```

Comment:
@artsyjian I think we needed to keep this copy - staging releases are failing with no templates: https://app.circleci.com/pipelines/github/artsy/positron/4081/workflows/1cfc8032-b2f2-4cf5-a876-97ef30377e3f/jobs/9090